### PR TITLE
Feature: data pipeline ext space

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: projects/elife.yaml [hunk 1]
```diff
@@ -2109,6 +2109,10 @@ data-pipeline:
                  deletion-policy: delete
             "{instance}-elife-data-pipeline-archive":
                  deletion-policy: retain
+    aws-alt:
+        prod:
+            ext:
+                size: 100 # GB
```
Comment (giorgiosironi) [inline projects/elife.yaml]: consider also `type: ssd` if it's not the default, if there are performance risks. If it's a staging area for files no worries, magnetic would be enough

Comment (lsh-0) [reply projects/elife.yaml]: I'm a bit fuzzy about what will pass through there to be honest. Nifi has backpressure and throughput controls and once it's configured it just hums along. Perhaps once there are more flows happening and competing for resources we can look at speeding up disk access, or bumping the instance type.

**Comment by giorgiosironi** [comment]
Can this be updated and merged?

**Comment by lsh-0** [comment]
absolutely, looks like it got forgotten about

**Pull request merged**